stdlib/float.mli

```diff
@@ -433,6 +433,19 @@ module Array : sig
       applied to the integers [0] to [n-1].
       @raise Invalid_argument if [n < 0] or [n > Sys.max_floatarray_length]. *)
 
+  val make_matrix : int -> int -> float -> t array
+(** [make_matrix dimx dimy e] returns a two-dimensional array
+   (an array of arrays) with first dimension [dimx] and
+   second dimension [dimy]. All the elements of this new matrix
+   are initially physically equal to [e].
+   The element ([x,y]) of a matrix [m] is accessed
+   with the notation [m.(x).(y)].
+
+   @raise Invalid_argument if [dimx] or [dimy] is negative or
+   greater than {!Sys.max_array_length}.
+   If the value of [e] is a floating-point number, then the maximum
```

Comment:
`e` is always a floating-point number in this function, and the max bound for `floatarray` is `Sys.max_floatarray_length`.